src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -2701,7 +2682,7 @@ window.qBittorrent.DynamicTable ??= (() => {
         },
         updateIcons: function() {
             // state_icon
-            this.rows.each(row => {
+            this.rows.forEach((row, rowId) => {
```

Comment:
In general, you probably should not use `forEach` any more and use `getRowItems()` (or other getters) instead.